app/helpers/Intersect.js

```diff
@@ -396,7 +390,7 @@ export async function checkIntersect(
     hasMigratedOldData = true;
   }
 
-  const result = await asyncCheckIntersect(healthcareAuthorities, bypassTimer);
+  const result = await asyncCheckIntersect(healthcareAuthorities);
   console.log(`[intersect] ${result ? 'completed' : 'skipped'}`);
```

Comment:
This shouldn't be able to skip now I think?  (It'll skip at background task level)